liquibase-core/src/main/java/liquibase/Liquibase.java

```diff
@@ -571,13 +574,16 @@ public void update(int changesToApply, Contexts contexts, LabelExpression labelE
             throws LiquibaseException {
         changeLogParameters.setContexts(contexts);
         changeLogParameters.setLabels(labelExpression);
+        Scope.getCurrentScope().addMdcValue(MdcKey.CHANGESET_LABEL_FILTER, labelExpression != null ? labelExpression.toString(): "");
```

Comment:
I see a duplicate of code -Can we move this two lines to a method and call that method instead?

Reply:
Updated!